src/game/physics.lua

```diff
@@ -327,8 +328,12 @@ function physics:platforms(object, dt)
 				elseif collision:bottom(object,platform)
 				and not collision:right(object,platform)
 				and not collision:left(object,platform) then
+					console:print('bottom side collision ')
```

Comment:
May be worth removing that console:print()

Reply:
ok